doc/adrs/030_add_a_link_representation.adoc

```diff
@@ -0,0 +1,54 @@
+= ADR-030 - Add a link representation
+
+== Context
+
+We need to have a batter integration with remote systems.
+For that, we want to be able to display links related to the currently selected element.
+We will thus introduce a new representation named link to display those links.
```

Comment:
I did that on purpose, with the experience of EEF in mind, technically nothing prevents those links from being internal or even from triggering side effects. We may use the label Hyperlink for the name of the representation on the frontend but I want to keep open some options since we can do more with them.